README.md

```diff
@@ -137,6 +151,10 @@ Similarly, third party delegation is safe because first parties are already able
 other means (such as link decoration), to achieve the same information sharing
 with third parties, in less convenient and performant ways.
 
+Further, the HTML delegation can only occur when the `<meta>` HTML tag is not injected
+via javascript, ensuring third parties cannot delegate hints in ways the first-party
```

Comment:
maybe "ensuring scripts cannot delegate hints in ways that relax the restrictions the top-level content author did not permit"?